src/imgtools/cli/dicomshow.py

```
@@ -0,0 +1,144 @@
+    table = Table(title=f"", box=None)
+    table.add_column("Keyword", justify="left", style="cyan", no_wrap=True)
+    table.add_column("Value", style="magenta")
+    
+    if pydicom:
+
+        try:
+            logger.info(f"Extracting tags from {dicom_file} (pydicom method)")
+            result: Any = dcmread(file_path, stop_before_pixels=True)
+        except Exception as e:
+            logger.error(f"Failed to read DICOM file {file_path}: {e}")
+            raise click.ClickException(f"Cannot read DICOM file: {e}")
```

Ruff (0.11.9)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)